managers.md

```diff
@@ -91,60 +71,37 @@ through managing a manager.
-- You set clear goals and expectations and work with your managers to set clear goals and expectations within their teams.
-- You provide continuous feedback on whether goals and expectations are being met and how team members can improve.
-- You are focused on identifying and supporting ways to help the members on your team learn and grow through clear career development goals and objectives.
-- You coach members of your team and help them grow and develop.
-- You are able to and might manage multiple teams and managers.
-- You work with the engineering leadership team, your teams, and corresponding stakeholders to set a successful strategy and vision for your functional areas.
+- You manage all the teams in an Organization through the support and direction of the team managers.
+- You manage EMs (or ICs directly during a transition). You have multiple instances of successful promos, including ICs with a successful transition to EM or Staff Engineer.
+- You advocate for headcount and budgets to be allocated to the entire Organization, using feedback from EMs and Sr EMs as guidance. You understand the needs other Organizations have, and escalate or adjust if those needs will impede your Organization’s needs. You understand the strategy for the Company, and ensure your Organization is structured and capacity balanced to support it.
+- You scout and attract talent. You build relationships with potential candidates. You help in building candidate pipelines. You runs org wide hiring initiatives. You make key hire calls.
+- You provide a larger org view on retention and strongly influence decisions that can help in this regard.
 
 ### Software Engineering & Design
 
-- You hold your team to the software engineering and design standards we have set for individual contributors.
-- You provide an environment (process, tools, goals, sprints, etc) that allows managers and individual contributors to do their best work.
-- You contribute to technical decision making and exemplify the same standards we set for individual contributors.
-- You work with your teams to help set software engineering and design standards.
-- You facilitate adoption of your team's standards across the broader engineering team.
+- You build technical experts within the teams who can continue building out best practices (e.g. tech leads). You ensure these technical experts have opportunities to help build technical skills across the Organization your teams fit into.
+- You are a role model for interpersonal skills, execution and results. You focus on empowering ICs and managers to take on role modeling (technical & interpersonal), and share examples of that role modeling across teams. You ensure technical experts are brought into conversation within and external to the team, and establish them as a subject matter experts (SME) to the entire organization.
```

Comment:
These need to get broken out into multiple lines and I think this line, or something like it, should be included.

>- You facilitate adoption of your team's standards across the broader engineering team.